jOOQ/src/main/java/org/jooq/impl/JoinTable.java

```diff
@@ -49,7 +49,6 @@
 import static org.jooq.Clause.TABLE_JOIN_OUTER_FULL;
 import static org.jooq.Clause.TABLE_JOIN_OUTER_LEFT;
 import static org.jooq.Clause.TABLE_JOIN_OUTER_RIGHT;
-import static org.jooq.Clause.TABLE_JOIN_PARTITION_BY;
```

Comment:
These imports are not needed in the jOOQ Open Source edition, but in order to preserve the exact same line numbers with the commercial distributions, we don't clean up unneeded imports.

I'll put these back again after merging.